.travis.yml

```diff
@@ -52,5 +52,10 @@ jobs:
       env:
         - E2E_KUBERNETES_VERSION=v1.12.0
 
+# safelist
+branches:
+  only:
+  - master
```

Comment:
Would this prevent PR builds? 🤔

Reply:
This should allow Travis to run PRs branches + `master` branch.